website/docs/guides/best-practices/how-we-mesh/mesh-3-implementation.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "Implementing your mesh plan"
+description: Getting started with dbt Mesh patterns
+hoverSnippet: Learn how to get started with dbt Mesh
+---
+
+## Implementing a dbt Mesh
+
+Let's examine an outline of steps to start implementing a dbt Mesh in your organization.
+
+### Research your current structure
+
+- Look at your selectors to figure out how people are grouping models right now.
+- Talk to teams about what sort of separation is naturally existing right now
+  - Are there various domains people are focused on?
+  - Are there various sizes, shapes, and sources of data that get handled separately (such as click event data)?
+  - Are there people focused on separate levels of transformation, such as landing and staging data or building marts?
+
+## Add groups and access
+
+Once you have a sense of some initial groupings, implement group and access permissions within a project.
+
+- Incrementally start building your jobs based on these groups (we would recommend in parallel to your production jobs until you’re sure about them) to feel out that you’ve drawn the lines in the right place.
```

Comment:
might be worth calling out this can/should be done in a QA deployment environment